parsl/monitoring/visualization/app.py

```diff
@@ -1,14 +1,15 @@
 from flask import Flask
 from parsl.monitoring.visualization.models import db
 import argparse
+import os
 
 
 def cli_run():
     """ Instantiates the Monitoring viz server
     """
     parser = argparse.ArgumentParser(description='Parsl visualization tool')
-    parser.add_argument('db_path', type=str,
-                        help='Database path in the format sqlite:///<absolute_path_to_db>')
+    parser.add_argument('db_path', type=str, default="sqlite:///{cwd}/monitoring.db".format(cwd=os.getcwd()),
+                        nargs="?", help='Database path in the format sqlite:///<absolute_path_to_db>')
```

Comment:
Why do we need `nargs="?"` here?

Reply:
`nargs="?"` makes the `db_path` parameter optional, so that it can be omitted and the default used: without it, that `db_path` parameter defaults to needing exactly one value.

That's different behaviour to the other arguments because the others are options beginning with `-`, but `db_path` is just taken from the main command line text.